### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ikhoon**
- Curator 5.1.0 -> 5.2.0
- Dropwizard 2.0.23 -> 2.0.24
- Dropwizard Metrics 4.2.2 -> 4.2.3
- Jackson 2.12.3 -> 2.12.4
- Jetty 9.4.42 -> 9.4.23
- GraphQL-Java 16.2 -> 17.1
- gRPC-Java 1.38.1 -> 1.40.0
  - protobuf-java 3.12.0 -> 3.17.2
  - protobuf-gradle-plugin 0.8.13 -> 0.8.17
- Kotlin 1.5.10 -> 1.5.21
  - kotlinx-coroutines-core 1.5.0 -> 1.5.1
- Logback 1.2.3 -> 1.2.5
- Micrometer 1.7.1 -> 1.7.3
- Netty 4.1.65 -> 4.1.67
  - tcnative-boringssl-static 2.0.39 -> 2.0.40
  - transport-native-io_uring 0.0.5 -> 0.0.8
- java-jwt 3.16.0 -> 3.18.1
- Reactor 3.4.7 -> 3.4.9
- RESTeasy 4.6.1 -> 4.7.1
- RxJava 3.0.13 -> 3.1.0
- scala-collection-compat 2.4.4 -> 2.5.0
- ScalaPB 0.11.4 -> 0.11.5
- ScalaPB json4s 0.11.1 -> 0.12.0
- SLF4J 1.7.31 -> 1.7.32
- Spring 5.3.8 -> 5.3.9
- Spring Boot 2.5.2 -> 2.5.3
- ZooKeeper 3.6.2 -> 3.6.3
- Build
  - Checkstyle 8.43 -> 8.45.1
  - Error Prone 2.7.1 -> 2.8.1
  - Eureka 1.10.15 -> 1.10.16
  - Finagle 21.6.0 -> 21.8.0
  - Gradle 7.1 -> 7.1.1
  - graphql-dgs-client 4.3.1 -> 4.5.1
  - graphql-kotlin 4.1.1 -> 5.0.0-alpha.3
  - groovy-xml 3.0.5 -> 3.0.8
  - jmh 1.32 -> 1.33
  - jsoup 1.13.1 -> 1.14.2
  - MUnit 0.7.26 -> 0.7.28
  - Proguard 7.1.0 -> 7.1.1
  - Reactor gRPC 1.0.1 -> 1.1.0
  - reactor-kotlin-extensions 1.1.3 -> 1.1.4
- Shaded
  - copmletable-futures 0.3.4 -> 0.3.5
- Miscellaneous
  - Add a missing Brotli's license
  - Add a workaround for a configuration error of gradle-proto-plugin
    - https://github.com/google/protobuf-gradle-plugin/issues/391

### CODE REVIEW

File: gradle/scripts/lib/java-rpc-proto.gradle [hunk 1]
```diff
@@ -105,6 +121,7 @@ configure(projectsWithFlags('java')) {
 
         // Make sure protoc runs before the resources are copied so that .dsc files are included.
         project.afterEvaluate {
+
```
Comment (trustin) [inline gradle/scripts/lib/java-rpc-proto.gradle:124]: Revert

**Review by trustin — COMMENTED** [review]
Nice! Let me wait for the build to pass before approving.

**Review by trustin — APPROVED** [review]
Many thanks, @ikhoon !

**Comment by ikhoon** [comment]
Opened a PR for the change of the Gradle script. https://github.com/line/gradle-scripts/pull/107

File: dependencies.yml [hunk 2]
```diff
@@ -365,10 +371,10 @@ net.shibboleth.utilities:
 #      (Switch to the right tag to find out the right version.)
 org.apache.curator:
   curator-recipes:
-    version: &CURATOR_VERSION '5.1.0'
+    version: &CURATOR_VERSION '5.2.0'
```
Comment (minwoox) [inline dependencies.yml:374]: Commit description is different? 😄

**Review by minwoox — APPROVED** [review]
👍 👍 👍

File: dependencies.yml (hunk 2 above)
Comment (ikhoon) [reply dependencies.yml:374]: Oops... 😱

Comment (ikhoon) [reply dependencies.yml:374]: '3.6'3' is a ZooKeeper version. Fixed.

**Comment by ikhoon** [comment]
DNS metrics tests were failed. It seems like there were changes in DNS exceptions. Let me take a look.

**Comment by ikhoon** [comment]
> DNS metrics tests were failed. It seems like there were changes in DNS exceptions. Let me take a look.

I reverted the Netty version to 4.1.66.
It seems like there is a bug in DNS resolver of the Netty 4.1.67.
- https://github.com/netty/netty/issues/11591
- https://github.com/netty/netty/pull/11415/files#diff-41d08a26f08dcb6f92085ad9bb565c833bbe0e0d425e0fe7147fcc51b56d164aR57-R64

**Pull request merged**